plenum/server/node.py

```diff
@@ -2082,9 +2075,14 @@ def primary_selected(self, instance_id):
         if instance_id == 0:
             # TODO: 0 should be replaced with configurable constant
             self.monitor.hasMasterPrimary = self.primaryReplicaNo == 0
-
-        if self.view_change_in_progress and self.all_instances_have_primary:
-            self.on_view_change_complete(self.viewNo)
+        if self.all_instances_have_primary:
+            # Doing this for CurrentState message
+            # This problem is faced because viewNo is set on view change start
+            view_no = self.viewNo
+            if self.primaryDecider:
```

Comment:
This would be True if the node is started. Why would `do_view_change_if_possible` be executed if node has not started?